beherabikram

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Bikram Behere",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
```

Comment:
this is the bio in the example docs, which is my bio so this should be removed completely:
```suggestion
```